.eslintrc.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  // See https://github.com/torchbox/eslint-config-torchbox for rules.
+  extends: 'torchbox',
```

Comment:
Should we use https://github.com/wagtail/eslint-config-wagtail instead?

Reply:
I think we should. @chris-lawton when we talked about it last week, this is what I meant -- the linting config that Wagtail itself uses.

Reply:
Updated!